### PR TITLE
Fix for issue https://github.com/Azure/azure-powershell/issues/4992 

### PR DESCRIPTION
**Pull request by darshanhs90**
Fix for issue https://github.com/Azure/azure-powershell/issues/4992 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj [hunk 1]
```diff
@@ -717,6 +717,9 @@
     <None Include="SessionRecords\Microsoft.Azure.Commands.Resources.Test.ScenarioTests.RoleAssignmentTests\RaGetByScope.json">
```
Comment (maddieclayton) [inline src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: @darshanhs90 Can you add a snippet to the changelog to ensure this is picked up for the release? Other than that the PR looks good.

**Comment by maddieclayton** [comment]
on demand here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/499/

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj (hunk 1 above)
Comment (darshanhs90) [reply src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: Updated the changelog

File: src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj [hunk 2]
```diff
@@ -717,6 +717,8 @@
...
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.Resources.Test.ScenarioTests.RoleAssignmentTests\RaGetByUPNWithExpandPrincipalGroups.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (cormacpayne) [inline src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj]: @darshanhs90 looks like you're missing a `</None>`

**Comment by saston2003** [comment]
The -ExpandPrincipalGroups does not work still as far as I can see.  I have just updated to the latest version of Azurerm powershell but there is no difference.  It used to work in previous versions of powershell, as per what has been posed by other people previously.

**Pull request merged**